diff --git a/tools/export_onnx.py b/tools/export_onnx.py
index 8703166..6486194 100644
--- a/tools/export_onnx.py
+++ b/tools/export_onnx.py
@@ -88,7 +88,7 @@ def main():
     logger.info("loading checkpoint done.")
     dummy_input = torch.randn(args.batch_size, 3, exp.test_size[0], exp.test_size[1])
 
-    torch.onnx._export(
+    torch.onnx.export(
         model,
         dummy_input,
         args.output_name,
@@ -97,6 +97,7 @@ def main():
         dynamic_axes={args.input: {0: 'batch'},
                       args.output: {0: 'batch'}} if args.dynamic else None,
         opset_version=args.opset,
+        dynamo=False,
     )
     logger.info("generated onnx model named {}".format(args.output_name))
 
